lib/Core/Site/Values/Content.php

```diff
@@ -283,4 +307,44 @@ private function getContentInfo()
 
         return $this->contentInfo;
     }
+
+    /**
+     * @return \Netgen\EzPlatformSiteApi\API\Values\Content
+     */
+    private function getOwner()
+    {
+        if ($this->owner === null) {
+            try {
+                $this->owner = $this->site->getLoadService()->loadContent($this->contentInfo->ownerId);
+            } catch (NotFoundException $e) {
+                $this->owner = false;
```

Comment:
Why assigning false? Null is much more meaningful there (i.e. "Nothing there"). A boolean value **is** an actual value, while null is a *non-value* :wink:

Reply:
In order not to repeat the query if we already concluded that there's no owner user, I think.

Reply:
Then better use return asap pattern (if user exists return it), and add a dedicated flag

Reply:
Yup, it's because it can happen that owner no longer exists, so it makes no sense to repeat loading if it was already done.

Updated as suggested in https://github.com/netgen/ezplatform-site-api/pull/51/commits/03952bc210937e9abc4649facec317120581b516, thanks.

Reply:
👌